From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: I2C: add module alias to lm75 driver

This is still only a test, and not ready for inclusion.


Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/hwmon/lm75.c |   13 +++++++++++++
 1 file changed, 13 insertions(+)

--- gregkh-2.6.orig/drivers/hwmon/lm75.c
+++ gregkh-2.6/drivers/hwmon/lm75.c
@@ -33,6 +33,19 @@
 static unsigned short normal_i2c[] = { 0x48, 0x49, 0x4a, 0x4b, 0x4c,
 					0x4d, 0x4e, 0x4f, I2C_CLIENT_END };
 
+static struct i2c_device_id id[] = {
+	{ 0x48 },
+	{ 0x49 },
+	{ 0x4a },
+	{ 0x4b },
+	{ 0x4c },
+	{ 0x4d },
+	{ 0x4e },
+	{ 0x4f },
+	{ },
+};
+MODULE_DEVICE_TABLE(i2c, id);
+
 /* Insmod parameters */
 I2C_CLIENT_INSMOD_1(lm75);
 
